### PR TITLE
Fix Airbrake initializer

### PR DESCRIPTION
**Pull request by steventux**

### CODE REVIEW

**Comment by sihugh** [comment]
Hurrah!

File: config/initializers/airbrake.rb [hunk 1]
```diff
@@ -1,5 +1,12 @@
...
-  config.development_environments << "production"
+  if ENV.has_key?("ERRBIT_API_KEY")
+    config.api_key          = ENV["ERRBIT_API_KEY"]
+    config.host             = "errbit.#{ENV['GOVUK_APP_DOMAIN']}"
+    config.secure           = true
+    config.environment_name = ENV['ERRBIT_ENVIRONMENT_NAME']
+  else
+    # Adding "production" to the development environments causes Airbrake not
+    # to attempt to send notifications.
+    config.development_environments << "production"
```
Comment (tijmenb) [inline config/initializers/airbrake.rb]: We don't have this in the other airbrake initialisers. Do you know what it's for?

Comment (chao-xian) [inline config/initializers/airbrake.rb]: @tijmenb you added your review comment at the same minute @sihugh merged. Is this ok to deploy? Cc @steventux

Comment (steventux) [inline config/initializers/airbrake.rb]: This was originally the 'development' config for Airbrake (ie. for running locally) in real deployments this config would have been overwritten. I struggled to understand what it does but _I think_ it's to suppress Airbrake when running RAILS_ENV=production locally.
Maybe it should just go.

**Pull request merged**